src/pages/RegistrationForm.tsx

```diff
@@ -120,8 +127,9 @@ function RegistrationForm({ isStudent }: { isStudent: boolean }) {
 
             console.log(e);
             return false;
-          }
+          }          
```

Comment:
And this

Reply:
Have a look now..

Reply:
Hopefully, no more extra spaces in the latest commit